docs/data-sources/as_group.md

```diff
@@ -0,0 +1,140 @@
+
+* `scaling_group_name` - (Optional, String) Specifies the AS group name. Fuzzy search is supported.
+
+* `scaling_configuration_id` - (Optional, String) Specifies the AS configuration ID, which can be obtained using
+  the API for listing AS configurations.
+
+* `status` - (Optional, String) Specifies the AS group status. The options are as follows:
+  - **INSERVICE**: indicates that the AS group is functional.
+  - **PAUSED**: indicates that the AS group is paused.
+  - **ERROR**: indicates that the AS group malfunctions.
+  - **DELETING**: indicates that the AS group is being deleted.
+  - **FREEZED**: indicates that the AS group has been frozen.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the list.
+
+* `groups` - A list of AS groups.
+
+The `groups` block supports:
+
+* `scaling_group_name` - The name of the AS group.
+
+* `scaling_group_id` - The AS group ID.
+
+* `status` - The AS group status.
+
+* `scaling_configuration_id` - The AS configuration ID.
+
+* `scaling_configuration_name` - The AS configuration name.
+
+* `current_instance_number` - The number of current instances in the AS group.
+
+* `desire_instance_number` - The expected number of instances in the AS group.
+
+* `min_instance_number` - The minimum number of instances in the AS group.
+
+* `max_instance_number` - The maximum number of instances in the AS group.
+
+* `cool_down_time` - The cooldown period (s).
+
+* `lb_listener_id` - The ID of a typical ELB listener. ELB listener IDs are separated using a comma (,).
+
+* `lbaas_listeners` - The enhanced load balancers.
+  The [object](#lbaas_listener_object) structure is documented below.
+
+* `availability_zones` - The AZ information.
+
+* `networks` - The network information.
+  The [object](#network_object) structure is documented below.
+
+* `security_groups` - The security group information.
+  The [object](#security_group_object) structure is documented below.
+
+* `create_time` - The time when an AS group was created. The time format complies with UTC.
+
+* `vpc_id` - The ID of the VPC to which the AS group belongs.
+
+* `detail` - Details about the AS group. If a scaling action fails, this parameter is used to record errors.
+
+* `is_scaling` - The scaling flag of the AS group.
+
+* `health_periodic_audit_method` - The health check method.
+
+* `health_periodic_audit_time` - The health check interval.
+
+* `health_periodic_audit_grace_period` - The grace period for health check.
+
+* `instance_terminate_policy` - The instance removal policy.
+
+* `notifications` - The notification mode.
+
+* `delete_publicip` - Whether to delete the EIP bound to the ECS when deleting the ECS.
+
+* `delete_volume` - Whether to delete the data disks attached to the ECS when deleting the ECS.
+
+* `cloud_location_id` - This parameter is reserved.
```

Comment:
unclear description

Reply:
this parameter is useless, i will remove it.